checkout/src/main/java/com/checkout/tokenization/repository/TokenRepositoryImpl.kt

```diff
@@ -80,6 +82,8 @@ internal class TokenRepositoryImpl(
 
             val tokenResult = cardTokenizationNetworkDataMapper.toTokenResult(response)
 
+            riskSdkUseCase.execute(tokenResult)
```

Comment:
Maybe I have already asked but why you are not interested to collect data also for the cvv scenario and the google pay scenario?

Reply:
You did, my response was that If we can, that will be great. What we did in Frames iOS was to include the Risk SDK in "Checkout" (a more central place that "Frames iOS" uses for tokenisation for all scenarios). 

@fabio-insolia-cko is there a section like this on "Frames Android"?

Reply:
@precious-ossai-cko 
Dunno about IOS, what I can tell is that we have in the TokenRepositoryImpl we have an `private fun handleResponse(` that covers both standard and Google (so maybe it will be better to move the riskUseCase over there) and there is a `sendCVVTokenizationRequest` for the 3rd scenario, so is just a matter of adding the riskUseCase at line 132 to handle that token as well